pkg/rkeworker/docker.go

```diff
@@ -74,6 +75,12 @@ func runProcess(ctx context.Context, name string, p v3.Process, start bool) erro
 			}
 		} else {
 			matchedContainers = append(matchedContainers, container)
+			if forceRestart {
+				err := restart(ctx, c, container.ID)
```

Comment:
Nit:  we can write it in one line: `if err := restart(); err != nil {return err}`